gui-wm/hyprland/hyprland-9999.ebuild

```diff
@@ -32,6 +32,17 @@ HYPRPM_RDEPEND="
 	dev-vcs/git
 	virtual/pkgconfig
 "
+
+XWAYLAND_DEPEND="
+	X? (
+		x11-libs/libxcb:0=
+		x11-base/xwayland
+		x11-libs/xcb-util-errors
+		x11-libs/xcb-util-renderutil
```

Comment:
x11-libs/xcb-util-renderutil is not needed.

Reply:
I'm running a clean build to confirm that.